openedx/adg/lms/webinars/helpers.py

```diff
@@ -59,3 +59,20 @@ def send_cancellation_emails_for_given_webinars(cancelled_webinars):
             cancelled_webinar.start_time,
             list(webinar_email_addresses)
         )
+
+
+def send_webinar_registration_email(webinar, email):
+    """
+    Send webinar registration email to user.
+    Args:
+        webinar (Webinar): The webinar in which user is registered
+        email (str): User's email
+    Returns:
+        None
+    """
```

Comment:
Add a newline before `Args` and `Returns` in accordance with[ Google docstring guide ](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html)and [edx docstring guide](https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#docstrings).